service/src/main/java/uk/gov/hmcts/reform/fpl/service/LegalRepresentativeService.java

```diff
@@ -46,23 +46,20 @@ public void updateRepresentatives(Long caseId,
                     userId -> caseService.addUser(Long.toString(caseId),
                         userId,
                         userToBeAdded.getRole().getCaseRoles()),
-                    throwException(userToBeAdded)
+                    logUnableToFindUserFromOrganisationService(userToBeAdded)
```

Comment:
wonder if we should keep the exception for the adding part, so we don't add people that don't exist, but then log when we remove?

Reply:
oh yes, you're right.